user/libgnomekbd/template.py

```diff
@@ -0,0 +1,26 @@
+pkgname = "libgnomekbd"
+pkgver = "3.28.1"
+pkgrel = 0
+build_style = "meson"
+configure_args = ["-Dintrospection=false"]
```

Comment:
why no introspection?

Reply:
I copied the libgnomekbd template from #172, with a few modifications. I didn't touch this variable.

Reply:
yeah just remove this line, there's no reason to have introspection disabled as far as i can see